src/ui/layer_list_panel.css

```diff
@@ -69,6 +69,94 @@
   line-height: 18px;
 }
 
+.neuroglancer-layer-list-panel-item input[type="checkbox"] {
+  width: "1rem";
+  height: "1rem";
+  margin: "0.25rem";
+}
+
+.neuroglancer-layer-list-panel-color-value-wrapper {
+  position: relative;
+  width: 10px;
+  height: 10px;
+  padding: 5px;
+}
+
+.neuroglancer-layer-list-panel-color-value {
+  border-radius: 50%;
+  height: 10px;
+  width: 10px;
+}
+
+.neuroglancer-layer-list-panel-color-value.rainbow {
```

Comment:
I think we should consider refactor this to be like the layer bar css where it uses properties. It's a bit unharmonious between the two right now